### PR TITLE
SEO-173296 WPF Chart Getting Started

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed and image alt.

Regards,
Yvone.

### CODE REVIEW

**Review by MallikaRK — APPROVED** [review]
Approved

File: wpf/Charts/Getting-Started.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ chart.SecondaryAxis = secondaryAxis
 
 Run the project and check if you get following output to make sure you have configured your project properly to add [`SfChart`](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Charts.SfChart.html).
 
-![WPF Chart](Getting-Started_images/wpf-chart.png)
+![WPF Chart](getting-started_images/wpf-chart.png)
```
Comment (Girija-Rajendran) [inline wpf/Charts/Getting-Started.md:97]: Changed this name in folder

Comment (Yvone-Atieno) [reply wpf/Charts/Getting-Started.md:97]: the folder is changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**